src/components/GlobalText.tsx

```diff
@@ -7,15 +7,15 @@ interface GlobalTextProps {
   style?: TextStyle
   testID?: string
   onPress?: () => void
-  children: JSX.Element | string
+  children: React.ReactNode | string
```

Comment:
very low priority but how do we feel about importing node from `React` as we are with the `FunctionComponent` ?